### PR TITLE
Issue 12. Removing return

### PR DESCRIPTION
**Pull request by beatrizsanchezt**
Fixing issue https://github.com/12358-lab/my-portfolio/issues/12

### CODE REVIEW

File: src/navigation/sidebar.js [hunk 1]
```diff
@@ -72,7 +72,7 @@ const SideBar = () => {
       items={sideNav}
       style={{ width: "192px" }}
       renderItem={({ href, children }) => {
```
Comment (nestor-diaz) [inline src/navigation/sidebar.js]: Is this working? It shouldn't be. You're removing the `return`, you need an inline body like so:

```js
renderItem={({ href, children }) => <CustomLink to={href}>{children}</CustomLink>}
```

Or

```js
renderItem={({ href, children }) => (<CustomLink to={href}>{children}</CustomLink>)}
```

File: src/navigation/SideBar/SideBar.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
+import React, { useState } from "react";
```
Comment (nestor-diaz) [inline src/navigation/SideBar/SideBar.js]: These changes aren't expected in this PR, isn't it?

**Review by nestor-diaz — CHANGES_REQUESTED** [review]

File: src/navigation/SideBar/SideBar.js (hunk 2 above)
Comment (beatrizsanchezt) [reply src/navigation/SideBar/SideBar.js]: Removed

File: src/navigation/sidebar.js (hunk 1 above)
Comment (beatrizsanchezt) [reply src/navigation/sidebar.js]: Fixed

**Review by nestor-diaz — APPROVED** [review]

**Pull request merged**